projects/demo/src/pages/recipes/phone/examples/2-ru-phone/template.html

```diff
@@ -13,3 +13,11 @@
         (blur)="onBlur()"
     />
 </tui-input>
+
+<ng-template #ruFlag>
+    <img
+        width="28"
+        alt="Flag of Russia"
+        [src]="'RU' | tuiFlag"
```

Comment:
Maybe it's better to use the UAE flag now?

Reply:
We need at least 1 input with russian phone number (for our e2e tests).